modules/ROOT/pages/_partials/commons/common-sgw-replication.adoc

```diff
@@ -160,6 +174,148 @@ include::{root-partials}replication-filters.adoc[]
 
 include::{root-partials}replication-channels.adoc[]
 
+
+[#anchor-auto-purge-on-revoke]
+=== Auto-purge on Channel Access Revocation
+
+.Breaking Change
+[CAUTION]
+--
+By default, when a user loses access to a channel all documents in the channel (that do not also belong to any of user’s other channels) are auto-purged from the local database (in devices belonging to the user). +
+_In release 2.x these documents will remain in the local database._
+--
+
+==== Behavior
+Users may lose access to channels in a number of ways:
+
+* User loses direct access to channel
+* User is removed from a role
+* A role that user belongs to is revoked access to channel
+
+In 3.x, by default, when a user loses access to a channel, the next Couchbase Lite Pull replication auto-purges all documents in the channel from local Couchbase Lite databases (on devices belonging to the user) *unless* they belong to any of the user’s other channels -- see: <<tbl-revoke-behavior>>.
+
+Documents that exist in multiple channels belonging to the user (even if they are not actively replicating that channel) are not auto-purged unless the user loses access to all channels.
+
+Users will be receive an `AccessRemoved` notification from the DocumentListener if they lose document access due to channel access revocation; this is sent regardless of the current auto-purge setting.
+Clients wanting to control whether to sync previous auto purged versions of documents (rather than pull down purged documents) must also move the documents out of all of the users' channels so they are not synced down again.
+
+.Behavior following access revocation
+[#tbl-revoke-behavior, cols="^1h,2a,2a", options="header"]
+|===
+
+2+|System State
+^|Impact on Sync
+
+.>h|Replication Type
+^.>h|Access Control on Sync Gateway
+^.>h|Expected behavior when _enable_auto_purge=true_
+
+|Pull only
+|User revoked access to channel.
+
+Sync Function includes `requireAccess(revokedChannel)`
```

Comment:
Just the fact that the user lost access to the channel will result in a revocation. `requireAccess` is not required for pull revocation.

Reply:
removed requireaccess